src/panel/disconnected/Disconnected.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from "react";
+import styled, { createGlobalStyle } from "styled-components";
+import image from "../../assets/icon.svg";
+
+export const Disconnected: FC = () => (
+  <>
+    <GlobalStyle />
+    <Container>
+      <Logo alt={"Urql Eagle"} src={image} />
+      <Header>Waiting for exchange</Header>
+      <Hint>Make sure {"you're"} using the Urql Devtools exchange!</Hint>
```

Comment:
Could be nice to add the link to the exchange here!

Reply:
This is a good shout!

I'm going to leave it out for now just because I want to avoid giving the user the impression that interaction is required on this page (for most, it's only going to be present for a second or so).

Maybe this is something we can look into though!